### ISSUE TITLE
Error on clap version when trying to install cli

### ISSUE THREAD

**Issue by jbertovic** [evestera/json_typegen#17]
Looks like a great tool.  I can go in and fix this by grabbing the source and changing it but thought I'd just note it here.  Let me know if you want me to try to create a PR (a bit new at all this).

when trying to use `cargo install json_typegen_cli`

```
>cargo install json_typegen_cli
    Updating crates.io index
  Downloaded json_typegen_cli v0.4.0
  Downloaded 1 crate (14.9 KB) in 0.89s
  Installing json_typegen_cli v0.4.0
error: failed to compile `json_typegen_cli v0.4.0`, intermediate artifacts can be found at `C:\Users\jas\AppData\Local\Temp\cargo-installRFMGPW`

Caused by:
  failed to select a version for the requirement `clap = "~2.19.0"`
  candidate versions found which didn't match: 2.33.3, 2.33.2, 2.33.1, ...
  location searched: crates.io index
required by package `json_typegen_cli v0.4.0`
```

**Comment by evestera** [comment]
Thank you for the report. I had not noticed that the clap version had been yanked. Will fix as soon as possible. If you want to try it in the meantime there are precompiled binaries under releases ( https://github.com/evestera/json_typegen/releases ).

**Comment by evestera** [comment]
Fixed with a21d643 / [release v0.4.1](https://github.com/evestera/json_typegen/releases/tag/v0.4.1).

Thank you again for the report, and let me know if you have any further issues. 🙂
